index.html

```diff
@@ -201,5 +201,88 @@ <h3 class="white">Opieka nad grobami</h3>
         </p>
       </div>
     </section>
+    <section id="speaker">
+      <div class="section-row">
+        <div class="section-white">
+          <p class="text-black">
+            Współpracujemy z doświadczonym Mówcą Pogrzebowym, specjalizującym się w prowadzeniu <strong>modlitw w kaplicy pogrzebowej</strong> 
+oraz organizacji <strong>świeckich ceremonii pogrzebowych</strong>. 
+          </p>
+        </div>
+        <div class="section-white">
+          <div class="section-right-align">
+            <button>
+              <p class="button-text">MÓWCA POGRZEBOWY</p>
+            </button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+
+
+    <section id="contact">
+      <div id="contact-container">
+ 
+          <h3 class="contact-header">Kontakt</h3>
+
+
+       <div id="contact-two-columns"> 
+
```

Comment:
Maybe we could get rid of the unnecessary empty lines and fix the indents? :)

Reply:
I had to install Beautify extension. Prettier doesn't work :(

Reply:
Sure, if it does the job for you then it's fine. :) Maybe indeed prettier would only work, if we had a repo with npm initialiazed, and that's not needed for now. :) The only thing that matters is that formatting is consistent, so any tool is fine. :)